provider/schema.json

```diff
@@ -0,0 +1,494 @@
+                                ]
+                            },
+                            "start_time": {
+                                "$id": "#/properties/data/properties/trips/items/properties/start_time",
+                                "type": "number",
+                                "description": "The time the trip began, expressed as a Unix Timestamp",
+                                "default": 0.0,
+                                "examples": [
+                                    1529968782.421409
+                                ]
+                            },
+                            "end_time": {
+                                "$id": "#/properties/data/properties/trips/items/properties/end_time",
+                                "type": "number",
+                                "description": "The time the trip ended, expressed as a Unix Timestamp",
+                                "default": 0.0,
+                                "examples": [
+                                    1531007628.3774529
+                                ]
+                            },
+                            "parking_verification_url": {
+                                "$id": "#/properties/data/properties/trips/items/properties/parking_verification_url",
+                                "type": "string",
+                                "description": "A URL to a photo (or other evidence) of proper vehicle parking",
+                                "default": "",
+                                "examples": [
+                                    "https://data.provider.co/parking_verify/1234.jpg"
+                                ],
+                                "pattern": "^(https://.*)$"
+                            },
+                            "standard_cost": {
+                                "$id": "#/properties/data/properties/trips/items/properties/standard_cost",
+                                "type": "integer",
+                                "description": "The cost, in cents, that it would cost to perform the trip in the standard operation of the System",
+                                "default": 0,
+                                "examples": [
+                                    500
+                                ]
+                            },
+                            "actual_cost": {
+                                "$id": "#/properties/data/properties/trips/items/properties/actual_cost",
+                                "type": "integer",
+                                "description": "The actual cost, in cents, paid by the customer of the mobility as a service provider",
+                                "default": 0,
+                                "examples": [
+                                    520
+                                ]
+                            }
+                        }
+                    }
+                },
+                "status_changes": {
```

Comment:
My understanding of the narrative description is that the JSON payloads are different between whether they hit the `trips` or `status_changes` endpoints. So having the JSON schema describe both of them at once seems like it is not the right thing. Instead, I would think that there would be a separate schema for each of the endpoints (maybe both deriving from a common base schema?).

Or am I misunderstanding things?

Reply:
No, it's one of the things I meant to put on my list above... definitely needs to be cleaned up. 

Reading about [combining schemas](https://json-schema.org/understanding-json-schema/reference/combining.html) got me thinking that something along those line might work. Just haven't thought about how to do it yet.